shadowsocks/config.go

```diff
@@ -8,8 +8,11 @@ import (
 	"os"
 	"sort"
 	"strings"
+	"sync"
 	"time"
 
+	"github.com/arthurkiller/shadowsocks-go/encrypt"
```

Comment:
Is it a good idea to import your personal registry here?
Since this package will be imported into the official package.

Reply:
If you have time to add unit test, the code should become more trustworthy~

Reply:
fixed